src/Symfony/Component/Scheduler/Generator/MessageContext.php

```diff
@@ -21,6 +21,7 @@
 final class MessageContext
 {
     public function __construct(
+        public readonly ?string $name,
         public readonly TriggerInterface $trigger,
         public readonly \DateTimeImmutable $triggeredAt,
         public readonly \DateTimeImmutable|null $nextTriggerAt = null,
```

Comment:
oops I let this slip (I can do a separate PR but that seem overkill):
```suggestion
        public readonly ?\DateTimeImmutable $nextTriggerAt = null,
```